README.adoc

```diff
@@ -69,26 +69,36 @@ Contributing
 
 Thank you for contributing to this repo. Please adhere to the following guidelines when creating a pull request:
 
-. Make your pull request against the _master_ branch (not develio) if you are contributing to the most recent release (currently
+. Make your pull request against the _master_ branch (not develop) if you are contributing to the most recent release (currently
   Cloud 10). The _master_ branch is protected and cannot be merged without review.
 
-. If you are contributing to a previous release, please see _maintenance/cloud_<RELEASENUMBER>_. This branch is also 
+. If you are contributing to a previous release, please see _maintenance/cloud_<RELEASENUMBER>_. This branch is also
   protected and cannot be merged without review.
 
 . Your contributions can be in ASCIIdoc `.adoc` or DocBook XML `.xml`.
 
+. Commit messages must conform to the following:
+.. Line lengths must be 80 characters or less.
+.. Commit body content is required.
+.. The subject line cannot start with "SOC".
+.. At the end of the subject line, there must be a bug reference or an
+   explanation for a missing bug reference:
+*** Bugzilla or Jira references: (bsc#0123456) or (SOC-0123456)
+*** Multiple Bugzilla or Jira references: (bsc#0123456),(SOC-0123456)
```

Comment:
Just saw that my original comment on multiple references read thus:

> multiple bug references: "(bsc#0123456)" or "(SOC-0123456)"

I should have said:

> multiple bug references: e.g. "(bsc#0123456, SOC-0123456)"

Sorry.

And yes, multiple references should work, according to the regex. What does not work is combinations of multiple non-references, e.g.: "(trivial, typo)" does not work.